DevCycle-iOS.xcodeproj/project.pbxproj

```diff
@@ -530,7 +530,7 @@
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
 				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
-				MACOSX_DEPLOYMENT_TARGET = 10.13;
+				MACOSX_DEPLOYMENT_TARGET = "$(RECOMMENDED_MACOSX_DEPLOYMENT_TARGET)";
```

Comment:
I still think we need these set to `10.13`

Reply:
reverted back, was changed to test something but its all back and tests passing